bin/i18n/sync-down.rb

```diff
@@ -27,7 +27,7 @@ def sync_down
         line.gsub!(/[\|\/\-\\][\b]/, '')
 
         # skip lines detailing individual file extraction
-        next if line.start_with? "Extracting: "
+        next if line.start_with?("Extracting: ") || line.starts_with?("Building ")
```

Comment:
nit: you could alternatively do
```suggestion
        next if line.start_with?("Extracting: ", "Building ")
```

Reply:
Oh that's better thanks!